starter-core/src/test/groovy/io/micronaut/starter/feature/database/DataJdbcSpec.groovy

```diff
@@ -91,48 +92,41 @@ class DataJdbcSpec extends ApplicationContextSpec  implements CommandOutputFixtu
         semanticVersionOptional.isPresent()
     }
 
-    void "test config"() {
-        when:
-        GeneratorContext ctx = buildGeneratorContext(['data-jdbc'])
-
-        then:
-        ctx.configuration.containsKey("datasources.default.url")
-        ctx.configuration.get("datasources.default.dialect") == "H2"
-
-        when:
-        ctx = buildGeneratorContext(['data-jdbc', 'postgres'])
-
-        then:
-        ctx.configuration.containsKey("datasources.default.url")
-        ctx.configuration.get("datasources.default.dialect") == "POSTGRES"
-
-        when:
-        ctx = buildGeneratorContext(['data-jdbc', 'mysql'])
-
-        then:
-        ctx.configuration.containsKey("datasources.default.url")
-        ctx.configuration.get("datasources.default.dialect") == "MYSQL"
-
-        when:
-        ctx = buildGeneratorContext(['data-jdbc', 'mariadb'])
-
-        then:
-        ctx.configuration.containsKey("datasources.default.url")
-        ctx.configuration.get("datasources.default.dialect") == "MYSQL"
-
-        when:
-        ctx = buildGeneratorContext(['data-jdbc', 'oracle'])
-
-        then:
-        ctx.configuration.containsKey("datasources.default.url")
-        ctx.configuration.get("datasources.default.dialect") == "ORACLE"
+    void "test config for #buildTool for feature #features"(BuildTool buildTool,
+                                      String dialect,
+                                      List<String> features) {
+        given:
+        Options options = new Options(null, null, buildTool)
 
         when:
-        ctx = buildGeneratorContext(['data-jdbc', 'sqlserver'])
+        GeneratorContext ctx = buildGeneratorContext(features, options)
 
         then:
-        ctx.configuration.containsKey("datasources.default.url")
-        ctx.configuration.get("datasources.default.dialect") == "SQL_SERVER"
+        if (buildTool == BuildTool.MAVEN) {
+            assert ctx.configuration.containsKey("datasources.default.url")
+        }
```

Comment:
we should check Gradle doesn't contain this key

Reply:
done